pennylane/devices/default_mixed.py

```diff
@@ -134,13 +135,25 @@ def _asarray(array, dtype=None):
         return res
 
     def __init__(
-        self, wires, *, r_dtype=np.float64, c_dtype=np.complex128, shots=None, analytic=None
+        self, wires, *, r_dtype=np.float64, c_dtype=np.complex128, shots=None, analytic=None, readout_prob=None
```

Comment:
Note: let's have a docstring entry for `readout_prob` on line 50, similar to `shots`, etc. It should follow the [PennyLane docstring standards](https://pennylane.readthedocs.io/en/stable/development/guide/documentation.html#docstrings) and can contain 1-2 formulas (if desirable) to mention what the readout probability simulates exactly.

Also, this docstring entry can indicate that we're awaiting a `float`.